public/app/features/expressions/components/Math.tsx

```diff
@@ -12,24 +14,143 @@ interface Props {
 }
 
 const mathPlaceholder =
-  'Math operations on one more queries, you reference the query by ${refId} ie. $A, $B, $C etc\n' +
-  'Example: $A + $B\n' +
-  'Available functions: abs(), log(), is_number(), round(), ceil(), floor(), is_inf(), is_nan(), is_null(), ';
+  'Math operations on one more queries, you reference the query by ${refId} ie. $A, $B, $C etc\n' + 'Example: $A + $B';
 
 export const Math: FC<Props> = ({ labelWidth, onChange, query }) => {
+  const [showHelp, setShowHelp] = useState<boolean>(true);
```

Comment:
You can consider using `useBoolean/useToggle` from `react-use` to avoid writing the `toggleShowHelp` function